alb_no_logs.tf

```diff
@@ -82,5 +86,5 @@ resource "aws_lb_listener" "frontend_https_no_logs" {
 resource "aws_lb_listener_certificate" "https_listener_no_logs" {
   listener_arn    = "${aws_lb_listener.frontend_https_no_logs.*.arn[lookup(var.extra_ssl_certs[count.index], "https_listener_index")]}"
   certificate_arn = "${lookup(var.extra_ssl_certs[count.index], "certificate_arn")}"
-  count           = "${var.logging_enabled ? 0 : var.extra_ssl_certs_count}"
```

Comment:
Generally in terraform if you start down the road of using locals to express conditionals (a good pattern for providing a simpler variable for long and complex statements), it's better to capture ALL the associated logic in one place - namely on the local itself. This makes for more maintainable code because you aren't having to mentally stitch together separate conditionals in different places of the code base to make sense of how it works. I'd much prefer this to be that way.